ultimate-battle-league.json

```diff
@@ -23,10 +23,10 @@
         "AdversityBot" :{"name": "AdversityBot", "makers": ["tarehart"]},
         "Gibbus" :{"name": "Gibbus", "makers": ["Enderjed"]},
         "Diablo" :{"name": "Diablo", "makers": ["TheBeard"]},
-        "Penguinbot" :{"name": "Penguinbot", "makers": ["_DaCoolOne_"]},
         "CodenameCryo" :{"name": "Codename Cryo", "makers": ["B1ue_F1re"]},
-        "Bowtie" :{"name": "Bowtie", "makers": ["_DaCoolOne_"]},
-        "Tuxedo" :{"name": "Tuxedo", "makers": ["_DaCoolOne_"]},
+        "Penguin" :{"name": "Penguin", "makers": ["_DaCoolOne_"], "notes": "Penguin is the team captain of Team Dappur. Penguin is an OG bot, grabbing his first title in his rookie year, but has not faired so well in 1v1 tournaments. He forged team Dappur with the hopes of grabbing another title. Some people have alleged that the inspiration for Team Dappur's name was a typo. These accusations are completely false, and the accusers shall be hanged at dawn."},
+        "Bowtie" :{"name": "Bowtie", "makers": ["_DaCoolOne_"], "notes", "Easily identifiable with his yellow and green Fennec, Team Dappur's newest addition, Bowtie, claims to be the snappiest dresser. Tuxedo would disagree. Bowtie was initially the sub on Team Dappur, but had to step up after the team dropped Beetle over attitude issues."},
```

Comment:
We need a `:` after "notes" instead of a comma. Credit to github diff highlighting for catching this one

Reply:
Yup, my mistake. Fixed